rider-fsharp/src/main/java/com/jetbrains/rider/plugins/fsharp/services/fsi/runScript/FSharpRunScriptConfigurationFactory.kt

```diff
@@ -0,0 +1,14 @@
+package com.jetbrains.rider.plugins.fsharp.services.fsi.runScript
+
+import com.intellij.execution.configurations.RunConfigurationSingletonPolicy
+import com.intellij.openapi.project.Project
+import com.jetbrains.rider.run.configurations.DotNetConfigurationFactoryBase
+import org.jetbrains.annotations.NotNull
+
+class FSharpRunScriptConfigurationFactory(type: FSharpScriptConfigurationType) :
+  DotNetConfigurationFactoryBase<FSharpScriptConfiguration>(type) {
+  override fun getId() = "F# Script"
```

Comment:
I would recommend having the _id_ less human-readable and without spaces. Say, `FSharpScript` is more appropriate, I'd say.